bigchaindb/config_utils.py

```diff
@@ -98,7 +99,12 @@ def file_config(filename=None):
 
     logger.debug('file_config() will try to open `{}`'.format(filename))
     with open(filename) as f:
-        config = json.load(f)
+        try:
+            config = json.load(f)
+        except ValueError as err:
+            raise exceptions.ConfigurationError(
+                'Failed to load configuration from `{}`, reason: {}'.format(filename, err)
```

Comment:
I tried your branch locally, and the error is like:
```
Failed to load configuration from `/home/vrde/.bigchaindb`, reason: Expecting value: line 1 column 59 (char 58)
```

A clearer wording would be:
```
Failed to parse the JSON configuration from `{}`. {}
```
in this way we highlight that the problem is on **parsing** a **JSON** file.